os/temp/temp_dir.go

```diff
@@ -75,3 +77,13 @@ func TempDirHere() (*TempDir, error) {
 
 	return NewTempDir(cwd, "scoot-tmp-")
 }
+
```

Comment:
I figured that this was so minimal that it would be preferable to keep this as a one-file package. WDYT?